src/test/run-selftest-pg

```diff
@@ -6,9 +6,15 @@
 
 BASE_INSTANCE="$1"
 TESTS="$2"
+if [ "$STELLAR_RNG_SEED" ]; then
```

Comment:
I like that plan; I've done that in the latest version.

I did preserve the ability to append `--all-versions` to the default parameters just by setting `ALL_VERSIONS=1`, since `travis-build.sh` uses that, and I didn't want either to change that or to risk messing up other users of it that I might not have discovered.